Likert.Rmd

```diff
@@ -0,0 +1,183 @@
+# Chart: Stacked Bar Chart (For Likert Data)
+
```

Comment:
Please add your name in the 3rd line so that everyone can know who made this contribution (the 2nd line should be blank). After adding your name, I can merge your branch. Thanks!😃